src/Chapter2.hs

```diff
@@ -731,7 +740,8 @@ the list with only those lists that contain a passed element.
 
 🕯 HINT: Use the 'elem' function to check whether an element belongs to a list
 -}
-contains = error "contains: Not implemented!"
+contains :: (Eq a, Num a) => a -> [[a]] -> [[a]]
```

Comment:
That's a very nice and polymorphic type signature :+1: The task description tells about numbers only to make it easier to write a type signature because we haven't explained typeclasses and constraints yet. So we expected to see a type signature `Int -> [[Int]] -> [[Int]]`. But your type signature is more general, and no harm in leaving it. Though, in such case, the `Num a` constraint is redundant. You can make your function even more general by leaving only `Eq a`. Note, that for the function implementation you don't need to have `Num`.